PortalFetch/crawler_download.py

```diff
@@ -244,8 +280,11 @@ def main():
         fillAdvanceSearch(driver)
         # Save searched courses
         html = saveResult(driver)
-        filename = parser.get('config', 'filename')
-        quarter = parser.get('config', 'quarter')
+        # get quarter and fileName based On quarter_value in crawler.config
```

Comment:
As mentioned, this should be changed to `get quarter and filename based on quarter_value in crawler.config`.
`fileName` -> `filename` because you are using `filename` as variable name in this function
`On` -> `on` because `O` should not be upper case in this context